tdrs-backend/tdpservice/data_files/test/factories.py

```diff
@@ -18,7 +18,7 @@ class Meta:
     extension = "txt"
     section = "Active Case Data"
     quarter = "Q1"
-    year = "2020"
+    year = 2020
```

Comment:
Model uses integer field, this should've never been a string. Ran into a type mismatch trying to decrement during our month conversions.